src/config/App.vue

```diff
@@ -92,11 +92,12 @@ export default defineComponent({
     if (store.state.auth.wantsToLogin) {
       updateLoginModal(true, false);
     }
-
+        
     watch(
       () => store.state.auth.user,
       async (newVal, oldVal) => {
         if (newVal && newVal != oldVal) {
+          document.body.classList.toggle('dark', newVal?.settings?.themes);
```

Comment:
let's call the variable `force_darkmode`
```suggestion
          document.body.classList.toggle('dark', newVal?.settings?.force_darkmode);
```